portfolio/src/main/webapp/script.js

```diff
@@ -34,3 +34,62 @@ function addRandomFact() {
   const factContainer = document.getElementById('fact-container');
   factContainer.innerText = fact; 
 }
+
+const size = 30;
```

Comment:
use upper case for constant. Also, as this is shared between different functions, consider renaming to a more specific name to indicate which size it refers to.